api/proto/wso2/discovery/service/websocket/frame_service.proto

```diff
@@ -56,6 +66,8 @@ message WebSocketFrameResponse {
   Code throttle_state = 1;
 
   int64 throttle_period = 2;
+
+  int32 apim_error_code = 3;
```

Comment:
Fix the bug where another connection made on throttled token/api, it should allow the connection to be made but no frame should be passed to the backend. Needs to achieve this via ext-authz-metadata

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2696/commits/7e3a5921553b6e94975f55cf44949c0863a035fe